src/main.rs

```
@@ -0,0 +1,129 @@
+use std::collections::HashMap;
+use std::env;
+use std::num::ParseIntError;
+use std::sync::Mutex;
+
+use actix_web::{server, App, HttpRequest, HttpResponse, Json};
+use sentry::integrations::failure::capture_error as capture_failure_error;
+use sentry::protocol::value::to_value;
+use sentry::User;
+use sentry_actix::SentryMiddleware;
+use serde::{Deserialize, Serialize};
+
+lazy_static::lazy_static! {
+    static ref HASHMAP: Mutex<HashMap<&'static str, u32>> = {
+        let mut inventory = HashMap::new();
```

Two changes here:
1. It is not possible to derive in a `lazy_static`. It simply has no effect. This construct declares a variable, not a new type. Usually, this would be an error, but the `lazy_static!` macro swallows this.
2. This actually caused a warning for me, since the variable name was uppercase.